pkg/api/types.go

```diff
@@ -1673,6 +1673,10 @@ const (
 // Capability represent POSIX capabilities type
 type Capability string
 
+// AllowAllCapabilities can be used as a value for the PodSecurityPolicy.AllowAllCapabilities
+// field and means that any capabilities are allowed to be requested.
+var AllowAllCapabilities Capability = "*"
```

Comment:
One nit. This should probably be in "pkg/apis/extensions" right? Since it's meant to be used with the PodSecurityPolicy types which are defined there.

Reply:
> This should probably be in "pkg/apis/extensions" right?

Probably? I'm either not sure what it should be.

Reply:
if it is related to PSP, it should be in the PSP API group (just like FSType All is)

Reply:
@php-coder do you need to run `./hack/update-all.sh -a` if the api was changed?